include/RAJA/util/plugins.hpp

```diff
@@ -38,7 +40,21 @@ callPostLaunchPlugins(PluginContext p) noexcept
   }
 }
 
+inline
+void
+callInitPlugins(PluginOptions p) noexcept
+{
+  for (auto plugin = PluginRegistry::begin(); 
+      plugin != PluginRegistry::end();
+      ++plugin)
+  {
+    (*plugin).get()->init(p);
+  }
+}
+
 } // closing brace for util namespace
 } // closing brace for RAJA namespace
 
+#include "RAJA/util/PluginInit.hpp"
```

Comment:
Rather that include the file down here, I think you should just move the `init` inside this file, and rename it `init_plugins`.